omnibus/CHANGELOG.md

```diff
@@ -1,13 +1,64 @@
 # Chef Server Changelog
 
-## Unreleased
+## 12.2.0 (2015-09-01)
 
-### chef-server
+### oc\_erchef
```

Comment:
Thanks, I just added this locally and a  chef-mover section with the fact a migration was added too.